ingress-nginx/values.yaml

```diff
@@ -1,8 +1,16 @@
 ingress-nginx:
   controller:
+    kind: "DaemonSet"
     admissionWebhooks:
       patch:
         enabled: false
+    config:
+      use-forward-headers: "true"
```

Comment:
Is this a typo for `use-forwarded-headers`? and if so: why do we want this? We shouldn't be trusting info from the client.